Directory.Build.targets

```diff
@@ -1,7 +1,10 @@
-﻿<Project>
+<Project>
   <PropertyGroup>
     <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == 'netcoreapp2.0' ">$(MicrosoftNETCoreApp20PackageVersion)</RuntimeFrameworkVersion>
     <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == 'netcoreapp2.1' ">$(MicrosoftNETCoreApp21PackageVersion)</RuntimeFrameworkVersion>
```

Comment:
When will we remove 2.0 and 2.1?

Reply:
aspnet/Universe#1153. Removing these was bogging down the process of getting the 2.2 changes in because the logic of when they were and were not needed across 50 repos was convoluted. Feel free to remove them on any repo you understand.